pkg/utils/semver.go

```diff
@@ -26,3 +26,20 @@ func SortVersions(vStrArr []string) error {
 	}
 	return nil
 }
+
+// IsNewVersion compares the plugin version and the installed version.
+func IsNewVersion(pluginVersionStr, installedVersionStr string) bool {
```

Comment:
Considering this function is under `utils` can we rename the variables used in the argument (as well as within the function) to be more generic?

Proposing something here but feel free to use something different as well.

```suggestion
func IsNewVersion(incomingVersionStr, existingVersionStr string) bool {
```